sdk/core/Azure.Core/tests/ETagTests.cs

```diff
@@ -8,12 +8,16 @@ namespace Azure.Core.Tests
 {
     public class ETagTests
     {
-        [Test]
-        public void StringRoundtrips()
+        [Theory]
```

Comment:
There does not seem to be tests for the ctor with boolean param. Though I really don't think we should have this ctor.

Reply:
It's private.

Reply:
Ah, sorry. I somehow missed that,